.github/workflows/codeql.yml

```diff
@@ -0,0 +1,95 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Can we inherit from a template workflow, instead of copy/paste? In ADO you can, and it is required there for compliance.